### PR TITLE
Bundle versions are persisted when they are installed

### PR DESCRIPTION
**Pull request by vanstee**
:warning: Based on https://github.com/operable/cog/pull/503

We're not currently handling multiple versions of bundles at the same time, but we are now keeping track of the version installed. If you try to install another version of a previously installed bundle, you'll receive an error.

Closes https://github.com/operable/cog/issues/98

### CODE REVIEW

**Comment by mpeck** [comment]
I think if we rebase this guy it will make it a bit easier to review.

File: test/cog/rule_test.exs [hunk 1]
```diff
@@ -5,7 +5,7 @@ defmodule RuleTest do
   alias Cog.Models.Rule
 
   setup do
-    bundle = Bundle.changeset(%Bundle{}, %{name: "test_bundle", config_file: %{}, manifest_file: %{}}) |> Repo.insert!
+    bundle = Bundle.changeset(%Bundle{}, %{name: "test_bundle", version: "0.0.1", config_file: %{}, manifest_file: %{}}) |> Repo.insert!
```
Comment (mpeck) [inline test/cog/rule_test.exs]: We don't actually need the manifest_file bit. That was removed earlier. Just missed removing from some tests. If you want to remove it cool, if not, we can do it later.

Comment (vanstee) [inline test/cog/rule_test.exs]: I'll go ahead and remove it real quick

**Comment by mpeck** [comment]
:+1: Looks good

**Pull request merged**